cardano_node_tests/tests/tests_conway/test_info.py

```diff
@@ -239,3 +239,96 @@ def test_info(
             votes=governance_utils.VotedVotes(cc=votes_cc, drep=votes_drep, spo=votes_spo),
             txhash=vote_txid,
         )
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_info_deposit_return_after_expiry(
```

Comment:
Can the existing `test_info` test be extended to check the deposit return? We aim to limit the overall testrun runtime.

Reply:
existing test_info has been extended to check deposit return as well.